src/scss/index.scss

```diff
@@ -0,0 +1,12 @@
+@import './theme/main.scss';
+
+@import './layout/normalize/normalize.scss';
+@import './layout/normalize/my-normalize.scss';
+
+@import './layout/grid/variables.scss';
+@import './layout/grid/mixins.scss';
+@import './layout/grid/grid.scss';
+
+@import './pages/home.scss';
+@import './slider.scss';
+@import './categories.scss';
```

Comment:
There is a mix of styling tools, you are using css files and styled-components, it is not recommended to do this because it can lead to collisions and, as the code base keeps growing, it becomes harder to maintain and debug the app